src/main/kotlin/net/ccbluex/liquidbounce/features/module/modules/world/scaffold/ModuleScaffold.kt

```diff
@@ -607,4 +632,65 @@ object ModuleScaffold : Module("Scaffold", Category.WORLD) {
         return hasBlockInMainHand
     }
 
+    @Suppress("unused")
+    private val onNetworkTick = handler<PlayerNetworkMovementTickEvent> { event ->
+        if (event.state == EventState.PRE) {
+            for(i in 1..expandLenght) {
+                val block = findBestBlockPlacementTarget(
+                    player.blockPos.add(
+                        (-sin(player.yaw.toRadians()) * i).toInt(),
+                        -(player.blockPos.y - placementY),
+                        (cos(player.yaw.toRadians()) * i).toInt()
+                    ),
+                    BlockPlacementTargetFindingOptions(
+                        listOf(Vec3i(0, 0, 0)),
+                        player.activeItem,
+                        CenterTargetPositionFactory,
+                        BlockPlacementTargetFindingOptions.PRIORITIZE_LEAST_BLOCK_DISTANCE,
+                        player.blockPos.add(
+                            (-sin(player.yaw.toRadians()) * i).toInt(),
+                            -(player.blockPos.y - placementY),
+                            (cos(player.yaw.toRadians()) * i).toInt()
+                        ).toVec3d()
+                    )
+                )
+                if (block != null) {
+                    if (expandRotationMode == expandRotations.Backwards) {
+                        RotationManager.aimAt(
+                            Rotation(player.yaw - 180, player.pitch),
+                            considerInventory = considerInventory,
+                            configurable = ScaffoldRotationConfigurable,
+                            provider = this@ModuleScaffold,
+                            priority = Priority.IMPORTANT_FOR_PLAYER_LIFE
+                        )
+                    }
+                    interaction.sendSequencedPacket(world) { sequence ->
+                        PlayerInteractBlockC2SPacket(
+                            Hand.MAIN_HAND,
+                            BlockHitResult(
+                                block.interactedBlockPos.toVec3d(),
+                                block.direction,
+                                block.interactedBlockPos,
+                                true
+                            ),
+                            sequence
+                        )
+                    }
```

Comment:
ahem i think you might be placing multiple blocks per tick here which certainly is not a good idea